src/test/test_verify.cpp

```
@@ -101,6 +122,20 @@ TEST_CASE("unsupported forms are rejected after unmarshal", "[unmarshal]") {
         REQUIRE(std::holds_alternative<std::string>(prog_or_error));
         REQUIRE_THAT(std::get<std::string>(prog_or_error), Catch::Matchers::ContainsSubstring("bad instruction"));
     }
+
+    SECTION("tail call chain depth above 33 is rejected") {
```

Cppcheck (2.19.0)

[performance] Function parameter 'other' should be passed by const reference.

(passedByValue)